db/migrate/20170405065305_remove_image_password_audit.rb

```diff
@@ -0,0 +1,16 @@
+class RemoveImagePasswordAudit < ActiveRecord::Migration
+  def up
+    Audited::Audit.where(:auditable_type => 'Image').where('audited_changes like ?', '%password%').find_each do |audit|
+      audit.audited_changes.delete("password")
+      if audit.audited_changes.empty?
+        audit.destroy
```

Comment:
so we also need to add more logic so that we don't ignore password attribute and we create custom audit message, something like we did for user password change